contrib/packaging/rpm/skydive.spec

```diff
@@ -33,6 +37,7 @@ Source0:        https://%{import_path}/releases/download/v%{source}/skydive-%{so
 BuildRequires:  systemd
 BuildRequires:  libpcap-devel libxml2-devel
 BuildRequires:  llvm clang kernel-headers
+BuildRequires:  selinux-policy-devel, policycoreutils-devel
```

Comment:
nit: this can be moved to %package selinux section

Reply:
not the buildrequires, they only apply to the main package, for the whole build